src/cheri_pte.sail

```diff
@@ -16,17 +16,24 @@ bitfield PTE_Bits : pteAttribs = {
 /*
  * Reserved PTE bits used by extensions on RV64.
  *
- * There are no such bits on RV32/SV32, so we hard-code both StoreCap=1
- * and LoadCap=1 (i.e., there is no ability to disable capability loads
+ * There are no such bits on RV32/SV32, so we hard-code both CapWrite=1
+ * and CapRead=1 (i.e., there is no ability to disable capability loads
  * from or stores to pages in the SV32 MMU).
  */
 type extPte = bits(10)
 
 bitfield Ext_PTE_Bits : extPte = {
-  StoreCap     : 9, /* Permit capability stores */
-  LoadCap      : 8, /* Permit capability loads */
+  CapWrite     : 9, /* Permit capability stores */
+  CapRead      : 8, /* Permit capability loads */
 }
 
+/*
+ * CapWrite     = 1,
+ * CapRead      = 1,
+ * bits 0 .. 7  = 0
+ */
+let default_sv32_ext_pte : extPte = 0b1110000000
```

Comment:
```suggestion
let default_sv32_ext_pte : extPte = 0b1100000000
```

Reply:
Good catch; capdirty snuck in a few commits early.